build.gradle

```diff
@@ -14,7 +14,7 @@ graphqlCodegen {
 	// all config options:
 	// https://github.com/kobylynskyi/graphql-java-codegen/blob/main/docs/codegen-options.md
 	outputDir = new File("$buildDir/generated")
-	packageName = "de.unistuttgart.iste.gits.template.dto" // TODO: change to your package name
+	packageName = "de.unistuttgart.iste.gits.generated.dto"
```

Comment:
we cannot have different packages for each repo for the generated code, if we want to use the dtos in the common utility.
Here, the three utility classes added all require dtos, so we need to use the same package in all microservices.